### ISSUE TITLE
Add directive blacklist

### ISSUE THREAD

**Issue by hugorodgerbrown** [yunojuno/django-csp-plus#6]
Some directives are not fully supported - the canonical example being Safari not supporting the `*-elem` and `*-attr` extensions to `script-src` and `style-src`.

This can cause issues for a multitude of reasons and a lot of confusion - as the way in which the fallback process works means that if a `script-src-elem` directive exists it will override the `script-src` directive, but only for user agents that support it. 

In short - there are some directives you should probably avoid to make your life simpler.
